torch/csrc/api/src/nn/modules/linear.cpp

```diff
@@ -67,6 +67,26 @@ Tensor LinearImpl::forward(const Tensor& input) {
 
 // ============================================================================
 
+BiasImpl::BiasImpl(const BiasOptions& options_) : options(options_) {
+  // NOLINTNEXTLINE(clang-analyzer-optin.cplusplus.VirtualCall)
+  reset();
+}
+
+void BiasImpl::reset() {
+  bias = register_parameter("bias", torch::randn(options.num_features()));
+}
+
+void BiasImpl::pretty_print(std::ostream& stream) const {
+  stream << std::boolalpha
+         << "torch::nn::Bias(num_features=" << options.num_features() << ")";
+}
+
+Tensor BiasImpl::forward(const Tensor& input) {
+  return F::bias(input, bias);
+}
+
+// ============================================================================
```

Comment:
nit: I think the definition for `reset_parameters()` declared in the header is missing here